actors/miner/tests/util.rs

```diff
@@ -763,73 +753,53 @@ impl ActorHarness {
         );
     }
 
+    // deprecated flow calling prove commit sector and then confirm sector proofs valid
+    // With FIP 0084 this is no longer realizable.  It is kept in tests
+    // 1) to ensure backwards compatibility between prove_commit_sector and prove_commit_sectors3
+    // 2) to make use of existing test coverage without significant change
```

Comment:
The behavior here has changed to using ProveCommit3, so this method is at least now poorly named. I suggest renaming it to reflect what it actually does. 

It looks like this has become essentially a wrapper for convenience around ProveCommitSectors3. Do you think it should be retained long term? The comment can be softened to reflect this. Or is there a follow-up task to remove or inline this?

Reply:
I'll rename and rework the comment.  Tracking cleanup in #1545